src/VisualStudio/Core/Def/Implementation/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -140,7 +140,7 @@ private enum GlobalNotificationState
             {
                 var projectTracker = vsWorkspace.GetProjectTrackerAndInitializeIfNecessary(Shell.ServiceProvider.GlobalProvider);
 
-                projectTracker.RegisterWorkspaceHost(host);
+                projectTracker.RegisterWorkspaceHost(host, neverPushProjects: true);
```

Comment:
didn't you get rid of this code from other PR?

Reply:
@heejaechang We were trying to see if we could do a more targeted fix still in 15.6, since my other PR is getting too big and risky.